core/src/main/java/hudson/model/ParametersDefinitionProperty.java

```diff
@@ -246,4 +238,14 @@ public String getIconFileName() {
     public String getUrlName() {
         return null;
     }
+
+    private class DefinitionsAbstractList extends AbstractList<String> {
```

Comment:
If we want to serialize it over the channel, maybe it's better to make the class static as well. Otherwise bits of the parent class will be serialized as well

Reply:
Wait, is this comment about the correct class? Is this one being serialized?